canonical-websites/www.ubuntu.com
# fix flavours page bugs

## Done

* breadcrumb not highlighted
* ampersand and caps in Budgie
* remove Edubuntu
* order flavours alphabetically
* s/Localizations/Localisations/

## QA

1. go to [/about/about-ubuntu/flavours](http://www.ubuntu.com-1516-about-ubuntu-fix.demo.haus/about/about-ubuntu/flavours)
2. see that the breadcrumb nav highlights
3. see the text changes

## Issue / Card

Fixes #1516
